main.go

```diff
@@ -68,16 +68,23 @@ func main() {
 
 	libp2pctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	p2pHost, _, err := CreateLibp2pHost(libp2pctx, tcpPort, wsPort, psk, privKey)
```

Comment:
Why the variable renaming? host is also a libp2p package ( "github.com/libp2p/go-libp2p/core/host") I know it's not used in this file but can lead to confusion.